docs/04-operation-guides/operations/obsv-02-adjust-loki.md

```diff
@@ -14,10 +14,13 @@ In your custom [Loki values.yaml](https://github.com/kyma-project/kyma/blob/main
 
 ## Adjust volume size
 
-If you're running out of volume (for example, because you increased the log retention period), you can expand the volume size.
+The way to change the volume size depends on whether you want to do it during the installation or later.
 
-1. In your custom [Loki values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/loki/values.yaml) file, enter the desired volume size as value for `persistence: size`.
-1. Additionally, expand the Persistent Volume Claims (PVC). For more information, see Kubernetes documentation on [Expanding Persistent Volumes Claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims).
+If you know already when you're installing Kyma that the default storage size isn't going to be enough, expand the Loki volume size for the initial deployment:
+In your custom [Loki values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/loki/values.yaml) file, enter the desired volume size as value for `persistence: size`.
```

Comment:
```suggestion
In your custom [Loki values.yaml](https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/loki/values.yaml) file, enter the desired volume size as the value for **persistence.size**.
```
As per [our guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md), values are formatted as `code`, parameters are formatted as **bold**; **persistence.size** is a parameter. Value is what the user is supposed to put there.
Also, when you refer to a parameter from a YAML, you use the dot notation, where each dot indicates one level deeper, if you know what I mean. 

See for example [this document](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-configuration-parameters/ac-01-application-connector-chart/).

Reply:
Oh boy, thanks!